bin/get-contracts.js

```diff
@@ -0,0 +1,38 @@
+const fs = require("fs");
+const dotenv = require("dotenv");
+const { exec } = require("child_process");
+dotenv.config();
+
+const CORE_EVM_CONTRACTS = [
```

Comment:
Hmm instead of hardcoding these I wonder if we could get them directly from the core-evm-package